projects/ion/src/lib/chip/chip.component.ts

```diff
@@ -18,8 +18,8 @@ export class ChipComponent {
   @Input('label') label!: string;
   @Input('disabled') disabled? = false;
   @Input('selected') selected? = false;
-  @Input() size?: Size = 'sm';
-  @Input() iconType?: IconType;
+  @Input() chipSize?: ChipSize = 'sm';
+  @Input() iconType?: IconType = 'pencil';
```

Comment:
```suggestion
  @Input() iconType?: IconType;
```
If you do this, an icon will always be rendered, even if user doesn't set iconType attribute to element.